espnet/nets/pytorch_backend/conformer/positionwise_feed_forward.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
```

Comment:
This file is still redundant.

Reply:
Thanks so much for your help. I keep this file because I make the activation function as an optional item, such as ReLU() and Swish() etc. Should I merge this difference to the Transformer's positionwise_feed_forward.py or just keep this file now?

Reply:
Since most lines are the same, you can slightly modify the original file.